sql/migrations/20200616191529_world.sql

```diff
@@ -0,0 +1,298 @@
+(934, 43, -9288.04, -2214.98, 62.9282, 100, 0, 0, 0),
+(934, 44, -9284.85, -2219.62, 63.3034, 100, 0, 0, 0),
+(934, 45, -9284.63, -2221.7, 63.3289, 100, 0, 0, 0),
+(934, 46, -9284.31, -2224.9, 63.3682, 100, 0, 0, 0),
+(934, 47, -9284.06, -2228.64, 63.411, 100, 0, 0, 0),
+(934, 48, -9283.2, -2246.79, 63.6757, 100, 0, 0, 0),
+(934, 49, -9281.5, -2257.96, 65.0833, 100, 0, 0, 0),
+(934, 50, -9279.75, -2263.68, 67.0197, 100, 0, 0, 0),
+(934, 51, -9276.29, -2271.59, 67.3239, 100, 0, 0, 0),
+(934, 52, -9273.32, -2269.94, 66.8647, 100, 0, 0, 0),
+(934, 53, -9271.99, -2267.63, 66.4996, 100, 0, 0, 0),
+(934, 54, -9271.01, -2264.53, 66.1423, 100, 0, 0, 0),
+(934, 55, -9270.74, -2249.5, 64.22, 100, 0, 0, 0),
+(934, 56, -9270.26, -2243.47, 64.0462, 100, 0, 0, 0),
+(934, 57, -9269.59, -2225.59, 64.0378, 100, 0, 0, 0),
+(934, 58, -9267.76, -2221.29, 64.0534, 100, 0, 0, 0),
+(934, 59, -9262.33, -2219.25, 63.9343, 100, 0, 0, 0),
+(934, 60, -9265.77, -2214.89, 64.0584, 100, 0, 0, 0),
+(934, 61, -9267.37, -2203.34, 64.0586, 100, 0, 0, 0),
+(934, 62, -9267.42, -2203.13, 64.0586, 100, 0, 0, 0),
+(934, 63, -9265.44, -2199.72, 64.0586, 100, 0, 0, 0),
+(934, 64, -9262.57, -2199.12, 64.0567, 100, 0, 0, 0),
+(934, 65, -9265.52, -2199.73, 64.0586, 100, 0, 0, 0),
+(934, 66, -9266.46, -2197.24, 63.977, 100, 0, 0, 0),
+(934, 67, -9267.15, -2190.66, 64.0898, 100, 0, 0, 0);
+
+
+--add effsee and hilary scripts
+UPDATE `creature_template` SET `ai_name` = "EventAI" WHERE entry = 8963;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (896301, 8963, 0, 1, 0, 100, 1, 1000, 15000, 150000, 180000, 896301, 896302, 0, 'effsee - speak');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (896301, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4627, 0, 0, 0, 0, 0, 0, 0, 0, 'effsee - speak');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (896302, 0, 68, 896201, 2, 8962, 10, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'hilary - speak');
+INSERT INTO `event_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (896201, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4628, 0, 0, 0, 0, 0, 0, 0, 0, 'hilary - speak');

```

Comment:
Script is already present in database. Quest: 3741 - Endevent

Reply:
Corrected, thanks for giving this a look over.